generate_models.py

```diff
@@ -197,159 +187,266 @@ def generate_load_timeseries(paths, param):
 #     - name: demand_urbs2015.csv
 #     - information: t, XX.Elec
     '''
+
+    timecheck('Start')
+
     # Sector land use allocation
-	# The land use coefficients found in the assumptions table are normalized over each sector, and the results are stored in the table 'sector_lu'
-    sector_lu = pd.read_excel(paths["assumptions"], sheet_name='Landuse', index_col=0, usecols=[0,2,3,4])
+    # The land use coefficients found in the assumptions table are normalized over each sector, and the results are
+    #  stored in the table 'sector_lu'
+    sector_lu = pd.read_excel(paths["assumptions"], sheet_name='Landuse', index_col=0, usecols=[0, 2, 3, 4])
     landuse_types = [str(i) for i in sector_lu.index]
-    sector_lu = sector_lu.transpose().div(np.repeat(sector_lu.sum(axis=0)[:,None],len(sector_lu),axis=1))
+    sector_lu = sector_lu.transpose().div(np.repeat(sector_lu.sum(axis=0)[:, None], len(sector_lu), axis=1))
     sec = [str(i) for i in sector_lu.index]
     sec_dict = dict(zip(sec, param["load"]["sectors"]))
-	
+
     # Share of sectors in electricity demand
     sec_share = pd.read_csv(paths["sector_shares"], sep=';', decimal=',', index_col=0)
+    stat = None
+
+    # Count pixels of each land use type and create weighting factors for each country:
+    # Population
+    stat_pop = pd.DataFrame.from_dict(zonal_stats(paths["Countries"], paths["POP"], 'population'))
+    stat_pop.rename(columns={'NAME_SHORT': 'Country', 'sum': 'RES'}, inplace=True)
+    stat_pop.set_index('Country', inplace=True)
 
-    if not (os.path.isfile(paths["load"] + 'df_sectors.csv') and os.path.isfile(paths["load"] + 'load_sector.csv') and os.path.isfile(paths["load"] + 'load_landuse.csv')):
```

Comment:
did you remove this line completely?

Reply:
I replaced the three .csv files by one .hdf file. The path is then just paths["load"]